reconcile/utils/quay_api.py

```diff
@@ -90,6 +101,33 @@ def add_user_to_team(self, user, team):
             raise RequestsException(r)
         return True
 
+    def create_or_update_team(self, team: str, role="member",
+                              description=None) -> None:
```

Comment:
Could you keep the same format with other methods and remove the team type and return type? I just want or keep consistency.

Reply:
You'd like me to remove the docstring and type hints?  My reasons for adding those were:

- The docstring provides a link to the Quay API docs which might be useful for quick references?  It also documents which exceptions can be raised.
- Gradually introducing the type hints can assist IDEs in auto-completion and prepare the code base for the opportunity to use tools like mypy, pyre, etc. in the future.

I understand this affects the consistency of the code, but it opens up the opportunity to improve the documentation of the other methods next time a change needs to be made to them.

Reply:
works for me